chi2/chinotype.py

```diff
@@ -380,23 +413,67 @@ def prepChi(self):
                     group by ccd
                 ) chicon
                 left join (
-                    select concept_cd, min(name_char) name
-                    from {2}.concept_dimension
+                    select concept_cd, min(name) name
+                    from (
+		      select c_basecode concept_cd,c_name name from {3}.{4}
+		      where ({5}) and {6}
```

Comment:
This sub-query becomes necessary because now there are two sources for codes and names: `CONCEPT_DIMENSION` and the ontology table.